study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/BuilderUtils.java

```diff
@@ -1,13 +1,25 @@
 package org.broadinstitute.ddp.studybuilder;
 
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.typesafe.config.Config;
 import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.activity.definition.template.Template;
 import org.broadinstitute.ddp.studybuilder.task.CustomTask;
+import org.broadinstitute.ddp.util.ConfigUtil;
+import org.broadinstitute.ddp.util.GsonPojoValidator;
+import org.broadinstitute.ddp.util.GsonUtil;
+import org.broadinstitute.ddp.util.JsonValidationError;
 
 /**
  * Various helpers for study-builder.
  */
 class BuilderUtils {
```

Comment:
Looks like its all Template related helpers, can we name it TemplateBuilderUtils ?

Reply:
There's other stuff here too, so I'll keep the name as-is.

Reply:
May be we should have named StudyBuilderUtils.java